### PR TITLE
Fix detekt warnings in format-common

### LINKED ISSUE #2029: [META] Clean up issues reported by Detekt
We recently introduced [Detekt](https://detekt.dev) to our CI suite and setup a [baseline](https://github.com/android-password-store/Android-Password-Store/tree/develop/detekt-baselines). Cleaning up the suppressed existing issues is a low effort, high value task that anyone can pick up without needing too much operational knowledge of how the app functions.

### PR DESCRIPTION
**Pull request by NewtonCesarRoncari**
Fixing some warnings reported by Detekt

I deleted the format-common file in detekt-baseline because running **./gradlew detektBaseline** the file stay empty

if possible, i would like this PR count for hacktoberfest

### CODE REVIEW

File: config/detekt/detekt.yml [hunk 1]
```diff
@@ -626,7 +626,7 @@ style:
     active: false
   ReturnCount:
     active: true
-    max: 2
+    max: 3
```
Comment (msfjarvis) [inline config/detekt/detekt.yml]: Don't change the configuration, if something cannot be refactored to fall under this limit then add a `@Suppress("ReturnCount")` on the method instead.

File: detekt-baselines/format-common.xml [hunk 2]
```diff
@@ -1,18 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<SmellBaseline>
```
Comment (msfjarvis) [inline detekt-baselines/format-common.xml:2]: Please let the file be empty.

File: format-common/src/main/kotlin/app/passwordstore/data/passfile/PasswordEntry.kt [hunk 3]
```diff
@@ -217,5 +217,7 @@ constructor(
         "secret:",
         "pass:",
       )
+    private const val THOUSAND = 1000
+    private const val THOUSAND_LONG = 1000L
```
Comment (msfjarvis) [inline format-common/src/main/kotlin/app/passwordstore/data/passfile/PasswordEntry.kt]: Prefer semantic names instead. Both these constants refer to 1 second, so there should just be one constant named `ONE_SECOND`. There are extension methods available in `delay` to also accept `kotlin.time.Duration` types so you can just do `delay(ONE_SECOND.seconds)` instead.

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt [hunk 4]
```diff
@@ -18,6 +18,13 @@ internal object Otp {
 
   private val BASE_32 = Base32()
   private val STEAM_ALPHABET = "23456789BCDFGHJKMNPQRTVWXY".toCharArray()
+  private const val BYTE_BUFFER_CAPACITY = 8
+  private const val END_INDEX_OFFSET = 4
+  private const val STEAM_GUARD_QUANTITY_DIGITS = 5
```
Comment (msfjarvis) [inline format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]:
```suggestion
  private const val STEAM_GUARD_DIGITS = 5
```

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt [hunk 5]
```diff
@@ -18,6 +18,13 @@ internal object Otp {
...
+  private const val MINIMUM_DIGIT = 6
+  private const val MAXIMUM_DIGIT = 10
```
Comment (msfjarvis) [inline format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]:
```suggestion
  private const val MINIMUM_DIGITS = 6
  private const val MAXIMUM_DIGITS = 10
```

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt [hunk 6]
```diff
@@ -59,17 +66,17 @@ internal object Otp {
         numDigits == null -> {
           return Err(IllegalArgumentException("Digits specifier has to be either 's' or numeric"))
         }
-        numDigits < 6 -> {
+        numDigits < MINIMUM_DIGIT -> {
           return Err(IllegalArgumentException("TOTP codes have to be at least 6 digits long"))
         }
-        numDigits > 10 -> {
+        numDigits > MAXIMUM_DIGIT -> {
           return Err(IllegalArgumentException("TOTP codes can be at most 10 digits long"))
```
Comment (msfjarvis) [inline format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]: Use the constants defined above in the error messages.

File: format-common/src/test/kotlin/app/passwordstore/data/passfile/PasswordEntryTest.kt [hunk 7]
```diff
@@ -194,8 +194,9 @@ class PasswordEntryTest {
 
   companion object {
 
+    private const val SECRET = "HXDMVJECJJWSRB3HWIZR4IFUGFTMXBOZ"
     const val TOTP_URI =
-      "otpauth://totp/ACME%20Co:john@example.com?secret=HXDMVJECJJWSRB3HWIZR4IFUGFTMXBOZ&issuer=ACME%20Co&algorithm=SHA1&digits=6&period=30"
+      "otpauth://totp/ACME%20Co:john@example.com?secret=$SECRET&issuer=ACME%20Co&algorithm=SHA1&digits=6&period=30"
```
Comment (msfjarvis) [inline format-common/src/test/kotlin/app/passwordstore/data/passfile/PasswordEntryTest.kt]: You should just suppress here.

**Review by msfjarvis — CHANGES_REQUESTED** [review]

File: config/detekt/detekt.yml (hunk 1 above)
Comment (NewtonCesarRoncari) [reply config/detekt/detekt.yml]: adjusted 👍

File: detekt-baselines/format-common.xml (hunk 2 above)
Comment (NewtonCesarRoncari) [reply detekt-baselines/format-common.xml:2]: adjusted 👍

File: format-common/src/main/kotlin/app/passwordstore/data/passfile/PasswordEntry.kt (hunk 3 above)
Comment (NewtonCesarRoncari) [reply format-common/src/main/kotlin/app/passwordstore/data/passfile/PasswordEntry.kt]: thanks for the tip, adjusted 👍

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt (hunk 4 above)
Comment (NewtonCesarRoncari) [reply format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]: adjusted 👍

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt (hunk 5 above)
Comment (NewtonCesarRoncari) [reply format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]: adjusted 👍

File: format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt (hunk 6 above)
Comment (NewtonCesarRoncari) [reply format-common/src/main/kotlin/app/passwordstore/util/totp/Otp.kt]: adjusted 👍

File: format-common/src/test/kotlin/app/passwordstore/data/passfile/PasswordEntryTest.kt (hunk 7 above)
Comment (NewtonCesarRoncari) [reply format-common/src/test/kotlin/app/passwordstore/data/passfile/PasswordEntryTest.kt]: adjusted 👍

**Review by msfjarvis — APPROVED** [review]

**Comment by msfjarvis** [comment]
Please run `./gradlew spotlessApply` to fix the codestyle violations.

**Comment by NewtonCesarRoncari** [comment]
@msfjarvis do is not necessary hacktoberfest label to count ?

**Comment by msfjarvis** [comment]
> @msfjarvis do is not necessary hacktoberfest label to count ?

The repository has the label.

**Pull request merged**